guidelines.rst

```diff
@@ -1,6 +1,12 @@
 PEP: 9999
 Title: Guidelines for Python's C API
 Author: Petr Viktorin <encukou@gmail.com>,
+        Guido van Rossum <guido@python.org>,
+        Victor Stinner <vstinner@python.org>,
+        Steve Dower <steve.dower@python.org>,
+        Erlend Egeberg Aasland <erlend@python.org>,
+        Serhiy Storchaka,
+        Michael Droettboom,
```

Comment:
Any reason why you're not including their email addresses?

Reply:
I only added addresses for people who already have them [on other PEPs](https://peps.python.org/pep-0000/#authors-owners).